src/static_functions.js

```diff
@@ -0,0 +1,32 @@
+function abortFetching() {
+    console.log('Now aborting');
+    controller.abort()
+}
+
+function getHeaders() {
+    return {
+         "Access-Control-Allow-Origin": "*", 'Access-Control-Allow-Methods':'GET,PUT,POST,DELETE,PATCH,OPTIONS'}
```

Comment:
nice, but better to write every key:value pair from a new line, like

```
return {
    "Access-Control-Allow-Origin": "*",
    'Access-Control-Allow-Methods':'GET,PUT,POST,DELETE,PATCH,OPTIONS'
}
```

and use quotes consistently